### PR TITLE
Update tag and reaction parameter docs

### PR DESCRIPTION
**Pull request by jhankin**
Fixes https://github.com/Raheem-ai/seed_web/issues/45.  @kidbombay let me know if I did this wrong.

### CODE REVIEW

**Comment by kidbombay** [comment]
For sanities sake, when doing any thing api related, make the fix first in one commit and make another commit after rake docs:generate
Hard to track down changes since it generates all docs.

File: spec/acceptance/incidents_spec.rb [hunk 1]
```diff
@@ -213,8 +213,8 @@
     parameter :incident_type_name, "OR Incident Type by Name"
     parameter :description, "What happened"
     parameter :location, "City and State"
-    parameter :reactions_list, "Reactions to incidents from approved Incdents"
-    parameter :tags_list, "Tags"
+    parameter :reactions_list, "Comma-delimited list of reactions"
```
Comment (kidbombay) [inline spec/acceptance/incidents_spec.rb]: Comma delimited list of existing reactions. 'stressed, scared, angry'

File: spec/acceptance/incidents_spec.rb [hunk 2]
```diff
@@ -213,8 +213,8 @@
...
+    parameter :tags_list, "Comma-delimited list of tags"
```
Comment (kidbombay) [inline spec/acceptance/incidents_spec.rb]: Same here
Comma delimited list of any tags. 'shooting, citation, arrest'

**Comment by jhankin** [comment]
@kidbombay updated, PTAL

**Comment by kidbombay** [comment]
PTAL?

**Comment by jhankin** [comment]
"Please take another look" -- sorry, maybe that's a Square-ism

**Comment by kidbombay** [comment]
Looks good!

**Pull request merged**